cmd/cloud-endpoints-controller/main.go

```diff
@@ -461,6 +519,11 @@ securityDefinitions:
 	return b.String(), nil
 }
 
+func makeDefaultOpenAPISpec(endpoint, target string, jwtAudiences []string) (string, error) {
```

Comment:
This function doesn't add much value. 

Can you refactor so that executeTemplate is only called once rather than in multiple places?